script.js

```diff
@@ -16,6 +16,171 @@ const initApp = () => {
   });
 };
 
+const projects = [
+  {
+    name: "Most-Post Storie's <br/> Gain+Glory",
+    description: 'Project 1 description',
```

Comment:
- [ ] [OPTIONAL] Kindly remove this text from the image, as it might not be so relevant.

- Your design
<img width="458" alt="image" src="https://github.com/AlyAtalla/AlyAtalla.github.io/assets/41163571/8af9ed7b-002e-4838-beb9-ae673b345678">

- Figma design
<img width="381" alt="image" src="https://github.com/AlyAtalla/AlyAtalla.github.io/assets/41163571/e138f0f5-987f-4866-9dba-fc8d68b429d6">

- Kindly fix the horizontal spacing between these items, to match the design 💯.